security-policies/bundle/compliance/cis_aws/rules/cis_2_1_5/data.yaml

```diff
@@ -123,7 +123,8 @@ metadata:
   impact: |-
     When you apply Block Public Access settings to an account, the settings apply to all AWS Regions globally. The settings might not take effect in all Regions immediately or simultaneously, but they eventually propagate to all Regions.
   default_value: ''
-  references: 1. https://docs.aws.amazon.com/AmazonS3/latest/user-guide/block-public-access-account.html
+  references: 1. 
```

Comment:
TODO: check this formatting

Reply:
Side effect of updating [ruamel.yaml](https://pypi.org/project/ruamel.yaml/). Tested E2E and no differences detected.